Sources/xcodeproj/PBXFileElement.swift

```diff
@@ -152,7 +152,10 @@ public extension PBXFileElement {
             guard let group = projectObjects.groups.first(where: { $0.value.childrenReferences.contains(reference) }) else { return sourceRoot }
             guard let groupPath = try group.value.fullPath(sourceRoot: sourceRoot) else { return nil }
             guard let filePath = self is PBXVariantGroup ? try baseVariantGroupPath() : path else { return groupPath }
-            return groupPath.appending(RelativePath(filePath))
+
+            // Swift PM generated pbxproj can have AbsolutePath
```

Comment:
I wouldn't add comments in the code unless it's necessary. In this case, one can read from the last line of the method that the path can be either absolute or relative.